tests/unittests/tests-hash_string/Makefile.include

```diff
@@ -0,0 +1,2 @@
+USEMODULE += hash_string
+
```

Comment:
The reason I added the extra new line is because all the other one's already had an extra line for instance [tests-timex MakeFile.include](https://github.com/RIOT-OS/RIOT/blob/master/tests/unittests/tests-timex/Makefile.include) , it is the same case with others like tests-crypto etc. So I intentionally added it here to.